src/components/Collapsible/tests/Collapsible.test.tsx

```diff
@@ -84,4 +85,43 @@ describe('<Collapsible />', () => {
       expect(collapsible.props()).toMatchObject({transition: {timingFunction}});
     });
   });
+
+  describe('onTransitionEnd', () => {
+    it('adds an isFullyClosed class to the collapsible onTransitionEnd if the event target is the collapsible div', () => {
+      const id = 'test-collapsible';
+      const collapsible = mountWithApp(<Collapsible id={id} open />);
+      collapsible.setProps({open: false});
```

Comment:
My opinion is that we should avoid setting props directly. You can create a composed component with a button and trigger on click to change the state internally.

Reply:
Updated! 
created a separate component in the test file to handle the prop updates by triggering a button